core/ecschema-metadata/src/UnitConversion/UnitConversion.ts

```diff
@@ -0,0 +1,83 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import { Constant, Unit } from "../ecschema-metadata";
+import { SchemaItemType } from "../ECObjects";
+import { Float } from "./Float";
+
+export class UnitConversion {
```

Comment:
mark entire class as @alpha, also add documentation ... this class and its methods are going to be one of the major points that users will interact with the api so it should all be documented

Reply:
Add some documentation to the class ... it won't be generated into docs yet (because it's alpha) but we will be moving it from alpha to beta as soon as we get some usage of it